packaging/nuget/package.ps1

```diff
@@ -0,0 +1,26 @@
+Push-Location $PSScriptRoot
+[Environment]::CurrentDirectory = $PWD
+
+$ProjectsDir = "..\..\src"
```

Comment:
Like I said I don't think this is a blocker, it's just the way paths are resolved in other scripts. If you prefer this way then go for it :)

One concern I do have though is what happens if one of the commands fails, and you exit before pop-location? Will that mess up other scripts?